job.go

```diff
@@ -78,6 +78,7 @@ type jobResponse struct {
 	LastUnstableBuild     jobBuild `json:"lastUnstableBuild"`
 	LastUnsuccessfulBuild jobBuild `json:"lastUnsuccessfulBuild"`
 	Name                  string   `json:"name"`
+	SubJobs               []job    `json:"jobs"`
```

Comment:
Maybe it is better to keep the field names of Go object and its JSON object are the same.